cypress/integration/internal/searchForLicence.spec.js

```diff
@@ -13,23 +13,41 @@ describe('search for licences as internal user', () => {
   it('user logs in and searches for a License', () => {
     // cy.visit to visit the URL
     cy.visit(Cypress.env('ADMIN_URI'));
-
-    // Enter the user name and Password
     cy.fixture('users.json').then(users => {
       cy.get('input#email').type(users.psc);
     });
     cy.get('#password').type(Cypress.env('DEFAULT_PASSWORD'));
-
-    // Click Sign in Button
     cy.get('.govuk-button.govuk-button--start').click();
 
     // assert once the user is signed in
     cy.contains('Licences, users and returns');
 
-    // search for a license
+    // search for a license by using Name
     cy.get('#query').type('Anglian').should('be.visible');
     cy.get('.search__button').click();
     cy.contains('Licences').should('be.visible');
+    cy.get('.govuk-table__body').contains('03/28/69/0034').should('be.visible');
+    cy.get('.govuk-table__body').contains('03/28/69/0034').click();
```

Comment:
This is **not** a test licence. Please remove this immediately. Gentle reminder that this is a public repository and we must never mix real data with the codebase.